src/js/keyboard.js

```
@@ -9,142 +9,120 @@ export class KeyboardSystem {
+                    break;
+                case 'line':
+                    this.axpObj.isLineMod = down;
+                    if (down && this.axpObj.isDrawing) this.axpObj.isLine = true;
+                    break;
             }
-            if (this.axpObj.isSHIFT) {
-                this.axpObj.isSHIFT = false;
+        };
+
+        window.addEventListener('blur', () => {
+            for (const [key, action] of Object.entries(this.activeModifiers)) {
```

ESLint

[error] 'key' is assigned a value but never used.

(no-unused-vars)